services/director-v2/src/simcore_service_director_v2/models/domains/comp_tasks.py

```diff
@@ -27,6 +27,9 @@ class ComputationTaskCreate(BaseModel):
         None,
         description="An optional set of nodes that must be executed, if empty the whole pipeline is executed",
     )
+    force_restart: Optional[bool] = Field(
```

Comment:
All these models, those being part of the director-v2 API ... we should start moving them to ``model/schemas``?

Reply:
okok ;)